javascript/mcp-server/src/index.ts

```diff
@@ -61,15 +63,15 @@ async function installForeverVM(options: { claude: boolean }) {
         config = JSON.parse(fileContent)
       } catch (error) {
         console.error('Failed to read or parse existing Claude config:', error)
-        return
+        process.exit(1)
       }
     }
 
     config.mcpServers = config.mcpServers || {}
 
     config.mcpServers.forevervm = {
       command: 'npx',
-      args: ['forevervm-mcp run'],
+      args: ['forevervm-mcp', 'run'],
```

Comment:
does this include a space? the previous version in one string worked for me

Reply:
it may have worked because the version of forevervm that ran ignored all of the arguments?

`['forevervm-mcp run']` is equivalent to running `npx "forevervm-mcp run"` in bash; `['forevervm-mcp', 'run']` is equivalent to running `npx forevervm-mcp run` in bash. The program itself (`npx` in this case) gets arguments as a list rather than a string; it's bash that turns the space-separated list into an array to pass to the program.